test/Cardano/FeeSpec.hs

```diff
@@ -679,12 +681,12 @@ instance Arbitrary RebalanceChangeOutputsData where
             , pure coinSum
             , choose (coinSum + 1, coinSum * 2)
             ]
-        pure $ RebalanceChangeOutputsData fee threshold coins
+        pure $ ReduceChangeOutputsData fee threshold coins
     shrink = genericShrink
```

Comment:
Using `genericShrink` here will eventually produce 0-valued coins, which may or may not be desirable. We should check this.

Reply:
Tested in 5533bf919b5b15dbc719bb3444b223fd4cbbc7f8.